client/data/reader/use-related-meta-by-tag.ts

```diff
@@ -104,6 +104,8 @@ const selectFromCards = ( response: {
 	related_tags: selectRelatedTags( response ),
 } );
 
+//TODO: this is the endpoint I need to use to get the related sites and tags for a given tag
+
```

Comment:
Do we still need this TODO for anything?  I think its already being used by the tags sidebar for related tags/sites.

Reply:
Oops no, this can be reverted. It was only a reminder.